### PR TITLE
Facebook comments feature and other fixes

### PR DESCRIPTION
**Pull request by sagarvora**
Fixed responsiveness of disclaimer, added facebook comments support in posts and moderators for the same, custom CSS to remove transition from anchor tag.

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -13,12 +13,11 @@
     </div>
 
     <div class="collapse navbar-collapse" id="navbar-collapse">
-      <ul class="nav navbar-nav">
-        <li><a href="/all-posts"><i class="fa fa-list"></i> All Posts</a></li>
-        <!--<li><a href="/gallery"><i class="fa fa-photo"></i> Photo Gallery</a></li>-->
-      </ul>
       <ul class="nav navbar-nav navbar-right">
-        <!--<li><a href="#"> <i class="fa fa-star"></i> Join GDG Baroda</a></li>-->
+        <li><a href="#"><i class="fa fa-calendar"></i> Events</a></li>
```
Comment (manan-jadhav) [inline _includes/navbar.html]: Inexistent link, please remove.

File: _layouts/post.html [hunk 2]
```diff
@@ -1,10 +1,27 @@
 ---
 layout: base
 ---
+<!-- Begin Facebook SDK -->
```
Comment (manan-jadhav) [inline _layouts/post.html]: Move this to the base layout please

File: _layouts/post.html [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+
+<!-- End Facebook SDK -->
+
 <div class="row">
   <div class="col-md-9">
     <div class="panel panel-body">
+
       {% include post_content.html %}
+
+      <div class="fb-comments" data-href="http://gdgbaroda.com{{ page.url }}" data-numposts="4" data-width="100%"></div>
```
Comment (manan-jadhav) [inline _layouts/post.html]: data-href is buggy

**Review by manan-jadhav — CHANGES_REQUESTED** [review]
Please keep "All Posts" as is, dont change it to "Blog"

**Comment by sagarvora** [comment]
Ok i'll make the changes

File: _includes/navbar.html (hunk 1 above)
Comment (sagarvora) [reply _includes/navbar.html]: shall I design a functionality to show events separately or shall we keep it as a part of 'events' category in blog?

Comment (manan-jadhav) [reply _includes/navbar.html]: We'll add stuff when we have enough content

**Comment by sagarvora** [comment]
Hey! Are any more changes required?

**Review by manan-jadhav — APPROVED** [review]

**Pull request merged**